lib/healthcheck/government_data_check.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Healthcheck
+  class GovernmentDataCheck
+    attr_reader :government_repo
+    def initialize
+      @government_repo = BulkData::GovernmentRepository.new
+    end
+
+    def name
+      :government_data_check
+    end
+
+    def status
+      return GovukHealthcheck::CRITICAL unless government_repo.cache_populated?
+      return GovukHealthcheck::WARNING if government_repo.cache_age > 21600
+
+      GovukHealthcheck::OK
+    end
+
+    def details
+      {
+        critical: "No government data availible",
+        warning: warning_details_content,
+      }
+    end
+
+    def enabled?
+      true
+    end
+
+  private
+
+    def warning_details_content
+      data_age = government_repo.cache_age.to_i
+      "Government data not refreshed in #{data_age / 3600} hours."
```

Comment:
If we're changing integers we could change the 3600 to 1.hour not sure if that's much of a reading win or not?

Reply:
Yeah i think it's worth it. Have adjusted, didn't realise you could be quite so fluid with durations and numbers.